specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2116 @@
+interface Entity {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("""
+Create or update an entity.
+Existing entity is matched using its unique guid if
+supplied or by its unique attributes eg: qualifiedName.
+Map and array of
+collections are not well supported. E.g., array<array<int>>, array<map<string,
+int>>.
+For each contact type, the maximum number of contacts is 20.
+""")
+  @route("/atlas/v2/entity")
+  @post
+  createOrUpdate is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Used to define the update behavior for business attributes when updating
+entities.
+""")
+      @query
+      businessAttributeUpdateBehavior?: BusinessAttributeUpdateBehavior;
+
+      @doc("""
+The collection where entities will be moved to. Only specify a value if you
+need to move an entity to another collection.
+""")
+      @query
+      collectionId?: string;
+
+      @doc("Atlas entity with extended information.")
+      @body
+      entity: AtlasEntityWithExtInfo;
+    },
+    EntityMutationResponse,
+    {},
+    ErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/prefer-csv-collection-format" "This is an existing Atlas API"
+  @doc("List entities in bulk identified by its GUIDs.")
+  @route("/atlas/v2/entity/bulk")
+  @get
+  listByGuids is Azure.Core.Foundations.Operation<
+    {
+      @doc("An array of GUIDs of entities to list.")
+      @query({format: "multi"})
+      guid: string[];
+
+      @doc("Whether to return minimal information for referred entities.")
+      @query
+      minExtInfo?: boolean;
+
+      @doc("Whether to ignore relationship attributes.")
+      @query
+      ignoreRelationships?: boolean;
+    },
+    AtlasEntitiesWithExtInfo,
+    {},
+    ErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("""
+Create or update entities in bulk.
+Existing entity is matched using its unique
+guid if supplied or by its unique attributes eg: qualifiedName.
+Map and array
+of collections are not well supported. E.g., array<array<int>>,
+array<map<string, int>>.
+For each contact type, the maximum number of contacts
+is 20.
+""")
+  @route("/atlas/v2/entity/bulk")
+  @post
+  bulkCreateOrUpdate is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+The collection where entities will be moved to. Only specify a value if you
+need to move an entity to another collection.
+""")
+      @query
+      collectionId?: string;
+
+      @doc("""
+Used to define the update behavior for business attributes when updating
+entities.
+""")
+      @query
+      businessAttributeUpdateBehavior?: BusinessAttributeUpdateBehavior;
+
+      @doc("An array of entities to create or update.")
+      @body
+      entities: AtlasEntitiesWithExtInfo;
+    },
+    EntityMutationResponse,
+    {},
+    ErrorResponse
```

Comment:
I would call out that the ErrorResponse shape is not compatible with error response shape defined in azure core  which will be referred in your RpcOperation one in getSampleBusinessMetadataTemplate. 

https://github.com/Azure/typespec-azure/blob/41bf47f801e64fbc35e03b8d1eb8022959834fa9/packages/typespec-azure-core/lib/foundations.tsp#L129